### PR TITLE
Add examples for custom automation triggers

### PR DESCRIPTION
**Pull request by WillRaphaelson**
Adds documentation for custom trigger configuration via the UI

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: docs/ui/automations.md [hunk 1]
```diff
@@ -63,6 +63,75 @@ For example, in the case of flow run state change triggers, you might expect pro
 !!! note "Work queue health"
     A work queue is "unhealthy" if it has not been polled in over 60 seconds OR if it has one or more late runs.
 
+Custom Triggers
+
+Custom triggers allow advanced configuration of the conditions on which a trigger executes its actions.
+
+![Viewing a custom trigger for automations for a workspace in Prefect Cloud.](../img/ui/automations-custom.png)
+ 
+For example, if you would only like a trigger to execute an action if it recieves two flow run failure events of a specific deployment within ten seconds, you could paste in the following trigger configuration:
```
Comment (serinamarie) [inline docs/ui/automations.md:72]:
```suggestion
For example, if you would only like a trigger to execute an action if it receives 2 flow run failure events of a specific deployment within 10 seconds, you could paste in the following trigger configuration:
```

File: docs/ui/automations.md [hunk 2]
```diff
@@ -63,6 +63,75 @@ For example, in the case of flow run state change triggers, you might expect pro
...
+
+```json
+{
+  "match": {
+    "prefect.resource.id": "prefect.flow-run.*"
+  },
+    "match_related": {
+     "prefect.resource.id": "prefect.deployment.70cb25fe-e33d-4f96-b1bc-74aa4e50b761"
```
Comment (serinamarie) [inline docs/ui/automations.md:80]:
```suggestion
  "match_related": {
    "prefect.resource.id": "prefect.deployment.70cb25fe-e33d-4f96-b1bc-74aa4e50b761"
```

File: docs/ui/automations.md [hunk 3]
```diff
@@ -63,6 +63,75 @@ For example, in the case of flow run state change triggers, you might expect pro
...
+  "expect": [
+    "prefect.flow-run.Completed"
+  ],
+  "posture": "Reactive",
+  "threshold": 2,
+  "within": 10
+}
+```
+
+Or, if you wanted your trigger to execute an action when a work queue didnt enter a healthy state from an unhealthy state after 30 seconds, you could paste in the following trigger configuration:
```
Comment (serinamarie) [inline docs/ui/automations.md:95]:
```suggestion
Or, if your work queue enters an unhealthy state and you want your trigger to execute an action if it doesn't recover within 30 minutes, you could paste in the following trigger configuration:
```

File: docs/ui/automations.md [hunk 4]
```diff
@@ -63,6 +63,75 @@ For example, in the case of flow run state change triggers, you might expect pro
...
+  "expect": [
+    "prefect.work-queue.healthy"
+  ],
+  "posture": "Proactive",
+  "threshold": 0,
+  "within": 1800
+}
+```
+
+Or, if you wanted your trigger to fire if your log write events passed a certain threshold, you could paste in the following trigger configuration:
```
Comment (serinamarie) [inline docs/ui/automations.md:118]:
```suggestion
Or, if you wanted your trigger to fire if your log write events passed a threshold of 100 within 10 seconds, you could paste in the following trigger configuration:
```

**Comment by serinamarie** [comment]
Nice! I added some suggestions for consistency/clarity but otherwise looks great 🙂

File: docs/ui/automations.md [hunk 5]
```diff
@@ -63,6 +63,75 @@ For example, in the case of flow run state change triggers, you might expect pro
 !!! note "Work queue health"
     A work queue is "unhealthy" if it has not been polled in over 60 seconds OR if it has one or more late runs.
 
+Custom Triggers
```
Comment (tpdorsey) [inline docs/ui/automations.md:66]:
```suggestion
### Custom Triggers
```

File: docs/ui/automations.md [hunk 6]
```diff
@@ -63,6 +63,75 @@ For example, in the case of flow run state change triggers, you might expect pro
...
+  "expect": [
+    "prefect.flow-run.Completed"
+  ],
+  "posture": "Reactive",
+  "threshold": 2,
+  "within": 10
+}
+```
+
+Or, if your work queue enters an unhealthy state and you want your trigger to execute an action if it doesn't recover within 30 minutes, you could paste in the following trigger configuration:
```
Comment (tpdorsey) [inline docs/ui/automations.md:95]:
```suggestion
If your work queue enters an unhealthy state and you want your trigger to execute an action if it doesn't recover within 30 minutes, you could paste in the following trigger configuration:
```

File: docs/ui/automations.md [hunk 7]
```diff
@@ -63,6 +63,75 @@ For example, in the case of flow run state change triggers, you might expect pro
...
+  "expect": [
+    "prefect.work-queue.healthy"
+  ],
+  "posture": "Proactive",
+  "threshold": 0,
+  "within": 1800
+}
+```
+
+Or, if you wanted your trigger to fire if your log write events passed a threshold of 100 within 10 seconds, you could paste in the following trigger configuration:
```
Comment (tpdorsey) [inline docs/ui/automations.md:118]:
```suggestion
If you wanted your trigger to fire if your log write events passed a threshold of 100 within 10 seconds, you could paste in the following trigger configuration:
```

**Review by tpdorsey — APPROVED** [review]

**Pull request merged**